client/lib/impure-lodash/index.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import {
```

Comment:
yeah that's right. in left all non-mocked usages in tact. my thinking was that if someone runs into the problem they will start looking for other tests with these mocks.

you raise a good point and I will try to think some about the discoverability issue.